scl/Makefile.am

```diff
@@ -15,7 +15,8 @@ SCL_SUBDIRS	= \
 	cisco		\
 	loggly		\
 	logmatic	\
-	snmptrap
+	snmptrap	\
+	windowspatternparser
 SCL_CONFIGS	= scl.conf syslog-ng.conf
```

Comment:
Why is it not an official parser? Those can come in the same directory. Anyway, Ill try to talk to him IRL.